defaulting-capi-clusters/README.md

```diff
@@ -0,0 +1,46 @@
+# Defaulting of CAPI clusters with webhooks
+
+This RFC describes a pattern for defaulting of CAPI clusters by utilizing Kubernetes webhooks extensively.
+
+## Problem statement
+
+We want clusters to be created with defaulted values.
+These values should match the clusters in our current product reasonably well.
```

Comment:
link to https://intranet.giantswarm.io/docs/product/pdr/006_capi-product-guidelines-values/
 points feature parity and sensible defaults